internal-enrichment/reversinglabs-spectra-analyze/src/main.py

```diff
@@ -199,21 +200,29 @@ def _submit_file_for_analysis(self, stix_entity, opencti_entity, hash, hash_type
             file_mime_type = "None"
             is_archive = False
 
+            analysis_report = None
             try:
                 response = self.a1000client.get_detailed_report_v2(
                     sample_hashes=self.hash,
                     retry=False,
                 )
-            except Exception as err:
-                raise ValueError(
-                    f"{self.helper.connect_name}: Looks like the sample you are trying to access may not be available on your Spectra Analyze instance. On Spectra Analyze run fetch and analyze on the sample."
-                ) from err
-
-            if response.status_code == 200:
+                # If no exception is raised, assume the response is valid
                 analysis_report = json.loads(response.text)
-            else:
-                raise Exception(
-                    f"{self.helper.connect_name}: There was issue with getting report from Spectra Analyze. HTTP Status code {str(response.status_code)}"
+            except NotFoundError:
+                self.helper.log_info(
+                    f"{self.helper.connect_name}: Detailed analysis report not found. Falling back to classification result."
+                )
+            except Exception as err:
+                self.helper.log_info(
```

Comment:
```suggestions
                self.helper.connector_logger.info(
```
We've deprecated this way to log, in order to unify and add more info to openCTI.
Also, shouldn't you log an error or a warning here and bellow ?